packages/suite/src/support/messages.ts

```diff
@@ -9001,7 +9001,7 @@ export default defineMessages({
         id: 'TR_STAKE_WHAT_IS_STAKING',
         defaultMessage: 'What is staking?',
     },
-    TR_STAKE_NETWORK_STAKING_IS: {
+    TR_STAKE_STAKING_IS: {
```

Comment:
The mismatch is no longer an issue, just doing it for consistency.

Reply:
Please use `TR_STAKE_NETWORK_STAKING_IS` or update string in crowdin